script.embuary.info/resources/language/resource.language.de_de/strings.po

```diff
@@ -0,0 +1,145 @@
+# Embuary Info language file
+# Addon Name: Embuary Info
+# Addon id: script.embuary.info
+# Addon Provider: sualfred
+# Translators:
+# sualfred <su4lfred@gmail.com>, 2019
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Embuary Helper\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: 2019-08-08 13:02+0000\n"
+"Last-Translator: sualfred <su4lfred@gmail.com>, 2019\n"
+"Language-Team: German (https://www.transifex.com/sualfred/teams/80018/de/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: de\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgctxt "#32000"
+msgid "Error"
+msgstr "Fehler"
+
+msgctxt "#32001"
```

Comment:
Fix

Reply:
Sorry, I don't get what you mean. What's wrong?

Reply:
disregard, looks like this is how multiline is done in gettext